src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -241,6 +242,25 @@ public boolean simulateBlockBreak(Block block)
 		return !event.isCancelled();
 	}
 
+	/**
+	 * Simulates the player placing a block. This method will not place the block if the player is in adventure or
+	 * spectator mode.
+	 *
+	 * @param material The material of the location to set to
+	 * @param location The location of the material to set to
```

Comment:
Missing documentation for the return type. It returns a bool, but what does the bool mean exactly? Is it `true` if the block was placed and `false` if it was not placed?